packages/components/src/styles/components/accordion.scss

```diff
@@ -36,75 +78,100 @@ $hds-accordion-item-border-radius: 6px;
     }
   }
 
-  &.hds-accordion-item--contains-interactive {
+  &.hds-accordion-item--contains-interactive,
+  &.hds-accordion-item--is-static {
     box-shadow: var(--token-surface-base-box-shadow);
   }
 }
 
+.hds-accordion-item--type-flush {
+  --hds-accordion-item-focus-ring-offset: 4px;
+  box-shadow: 0 1px 0 0 var(--token-color-border-primary);
+
+  &:first-child {
+    box-shadow: 0 1px 0 0 var(--token-color-border-primary), 0 -1px 0 0 var(--token-color-border-primary);
+  }
+
+  &.hds-accordion-item--does-not-contain-interactive:not(.hds-accordion-item--is-static) > .hds-disclosure-primitive__toggle {
+    &:hover,
+    &.mock-hover {
+      .hds-accordion-item__toggle {
+        background: var(--token-color-surface-interactive-hover);
+      }
+    }
+
+    &:active,
+    &.mock-active {
+      .hds-accordion-item__toggle {
+        background: var(--token-color-surface-interactive-active);
+      }
+    }
+  }
+}
+
 // TOGGLE BLOCK
 
 .hds-accordion-item__toggle {
   position: relative;
   display: flex;
-  gap: 12px;
+  gap: var(--hds-accordion-item-gap);
   align-items: center;
-  padding:
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
-    12px; // by design
+  padding: var(--hds-accordion-item-toggle-padding) var(--hds-accordion-item-toggle-padding) var(--hds-accordion-item-toggle-padding) var(--hds-accordion-item-toggle-padding-left);
 }
 
 .hds-accordion-item__button {
   padding: 0;
 
   &:hover { cursor: pointer; }
 
-  // entire toggle area is interactive
-  &.hds-accordion-item__button--parent-does-not-contain-interactive {
-    @include hds-focus-ring-with-pseudo-element();
-    position: static;
-    margin: -1px 0;
-    color: var(--token-color-foreground-primary);
-    background: transparent;
-    border: 1px solid transparent;
-
-    // expand button target to cover entire AccordionItem Toggle block (depending on the `@containsInteractive/@parentContainsInteractive` argument)
-    &::after {
-      position: absolute;
-      display: block;
-      border-radius: $hds-accordion-item-border-radius;
-      content: "";
-      inset: 0;
+  .flight-icon {
+    @media (prefers-reduced-motion: no-preference) {
+      transition: transform 0.3s;
     }
   }
 
-  // only chevron button area is interactive
-  &.hds-accordion-item__button--parent-contains-interactive {
-    @include hds-button();
-    width: 24px;
-    height: 24px;
+  // we hide it instead of not rendering it at all, to preserve the visual layout of the item, without requiring special CSS workarounds to compensate for the missing element
+  .hds-accordion-item--is-static.hds-accordion-item & {
+    visibility: hidden;
+  }
+}
 
-    &:focus,
-    &.mock-focus {
-      @include hds-button-state-focus();
-    }
+// entire toggle area is interactive
```

Comment:
@alex-ju Out of curiosity, are you utilizing the ::before and ::after pseudo-elements to style the focus state of the accordion component?

Reply:
yes, we are using the `::before` pseudo-element for the focus states (in many of our components)

![Screenshot 2024-06-25 at 10 16 19](https://github.com/hashicorp/design-system/assets/788096/b72803c9-d877-4de7-ab27-1b85a004783c)

Reply:
Ah, okay, thank you! I appreciate the example image as well.